src/lib/config.js

```diff
@@ -149,6 +149,9 @@ class Config {
     }
     this.apiDocumentationEndpoints = RC.API_DOCUMENTATION_ENDPOINTS || false
     this.protocolVersions = getProtocolVersions(DEFAULT_PROTOCOL_VERSION, RC.PROTOCOL_VERSIONS)
+    this.instrumentationMetricsDisabled = RC.INSTRUMENTATION.METRICS.DISABLED
```

Comment:
I observed many config parameters like this, where RC library is passing the value as string instead of boolean if we set the value through environment variable. Can we handle that here by something like this ?https://github.com/mojaloop/central-ledger/blob/274e446957b8f43028eab3d7a8242c7fbeb387c7/src/lib/config.js#L13****

Reply:
good catch. done.